web3_proxy/src/frontend/mod.rs

```diff
@@ -169,6 +169,14 @@ pub async fn serve(
         .route("/user", post(users::user_post))
         .route("/user/balance", get(users::payment::user_balance_get))
         .route("/user/deposits", get(users::payment::user_deposits_get))
+        .route(
+            "/user/stripe/deposits",
```

Comment:
Should we put the get and the post on the same route?

Reply:
hmm good point actually, let me change that later; works for me, also decreases naming complexity

Reply:
changed